apps/main/src/dex/store/createCreatePoolSlice.ts

```diff
@@ -1136,8 +1148,12 @@ const createCreatePoolSlice = (
           try {
             const coinAddresses = coins.map((coin) => coin.address)
             const assetTypes = coins.map((coin) => coin.ngAssetType)
-            const oracleAddresses = coins.map((coin) => (coin.ngAssetType === 1 ? coin.oracleAddress : zeroAddress))
-            const oracleFunctions = coins.map((coin) => (coin.ngAssetType === 1 ? coin.oracleFunction : '0x00000000'))
+            const oracleAddresses = coins.map((coin) =>
+              coin.ngAssetType === NG_ASSET_TYPE.ORACLE ? coin.oracleAddress : zeroAddress,
+            )
+            const oracleFunctions = coins.map((coin) =>
+              coin.ngAssetType === NG_ASSET_TYPE.ORACLE ? coin.oracleFunction : '0x00000000',
```

Comment:
what is 0x00000000 :scream: 
```suggestion
              coin.ngAssetType === NG_ASSET_TYPE.ORACLE ? coin.oracleFunction : undefined,
```